internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/ReportHandler.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *
+ *   Copyright (c) 2019, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+package org.wso2.security.tools.scanmanager.scanners.qualys.handler;
+
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.SftpException;
+import org.apache.commons.compress.archivers.ArchiveException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.security.tools.scanmanager.common.model.LogType;
+import org.wso2.security.tools.scanmanager.scanners.common.ScannerConstants;
+import org.wso2.security.tools.scanmanager.scanners.common.exception.ScannerException;
+import org.wso2.security.tools.scanmanager.scanners.common.util.CallbackUtil;
+import org.wso2.security.tools.scanmanager.scanners.common.util.ErrorProcessingUtil;
+import org.wso2.security.tools.scanmanager.scanners.common.util.FileUtil;
+import org.wso2.security.tools.scanmanager.scanners.qualys.QualysScannerConstants;
+import org.wso2.security.tools.scanmanager.scanners.qualys.config.QualysScannerConfiguration;
+import org.wso2.security.tools.scanmanager.scanners.qualys.model.ScanContext;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This method handles the report generation task.
+ */
+public class ReportHandler {
+
+    private static final Log log = LogFactory.getLog(ReportHandler.class);
+    String[] reportTypes = { QualysScannerConstants.PDF_TYPE, QualysScannerConstants.XML_TYPE,
+            QualysScannerConstants.HTML_BASE64_TYPE, QualysScannerConstants.CSV_V2_TYPE };
+    QualysScanHandler qualysScanHandler;
+
+    public ReportHandler(QualysScanHandler qualysScanHandler) {
+        this.qualysScanHandler = qualysScanHandler;
+    }
+
+    /**
+     * This method is responsible for tasks relevant to report.
+     *
+     * @param scanContext ScanContext
+     * @return true if report is suceesfully uploaded
+     * @throws ScannerException Error occurred while executing report handler.
```

Comment:
Follow the same format in other similar places.
```suggestion
     * @throws ScannerException Error occurred while executing report handler
```